src/controllers/remove.director.check.answers.controller.ts

```diff
@@ -50,17 +50,22 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
       }
     }
 
+    //Check if corporate director and format name based on corporate or individual director
     var corporateDirector = false;
+    let directorName = "";
```

Comment:
can we use  formatDirectorNameForDisplay() here too?

Reply:
I have moved the formatDirectorNameForDisplay() method into utils.format, and it's now being used in both remove.director.controller and remove.director.check.answers.controller